atr/get/distribution.py

```diff
@@ -66,55 +68,116 @@ async def list_get(session: web.Committer, project: str, version: str) -> str:
+async def delete_get(session: web.Committer, project: str, version: str) -> web.WerkzeugResponse:
+    # Redirect to the list page instead of showing a separate confirmation page
+    return await session.redirect(list_get, project=project, version=version)
+
+
 @get.committer("/distribution/record/<project>/<version>")
 async def record(session: web.Committer, project: str, version: str) -> str:
-    form = await shared.distribution.DistributeForm.create_form(data={"package": project, "version": version})
-    fpv = shared.distribution.FormProjectVersion(form=form, project=project, version=version)
-    return await shared.distribution.record_form_page(fpv)
+    await shared.release_validated(project, version, staging=False)
+
+    block = htm.Block()
+    shared.html_nav_phase(block, project, version, staging=False)
+
+    block.h1["Record a manual distribution"]
+    block.p[
+        "Record a distribution of ",
+        htm.strong[f"{project}-{version}"],
+        " using the form below.",
+    ]
+    block.p[
+        "You can also ",
+        htm.a(href=util.as_url(list_get, project=project, version=version))["view the distribution list"],
+        ".",
+    ]
+
+    # Render the distribution form
+    form_html = form.render(
+        model_cls=shared.DistributeForm,
+        submit_label="Record distribution",
+        action=util.as_url(post.distribution.record_post, project=project, version=version),
+        defaults={"package": project, "version": version},
+    )
+    block.append(form_html)
+
+    return await template.blank("Record Manual Distribution", content=block.collect())
 
 
 @get.committer("/distribution/stage/<project>/<version>")
 async def stage(session: web.Committer, project: str, version: str) -> str:
-    form = await shared.distribution.DistributeForm.create_form(data={"package": project, "version": version})
-    fpv = shared.distribution.FormProjectVersion(form=form, project=project, version=version)
-    return await shared.distribution.record_form_page(fpv, staging=True)
+    await shared.release_validated(project, version, staging=True)
+
+    block = htm.Block()
+    shared.html_nav_phase(block, project, version, staging=True)
+
+    block.h1["Record a staging distribution"]
+    block.p[
+        "Record a distribution of ",
+        htm.strong[f"{project}-{version}"],
+        " using the form below.",
+    ]
+    block.p[
+        "You can also ",
+        htm.a(href=util.as_url(list_get, project=project, version=version))["view the distribution list"],
+        ".",
+    ]
+
+    # Render the distribution form
+    form_html = form.render(
+        model_cls=shared.DistributeForm,
+        submit_label="Record distribution",
+        action=util.as_url(post.distribution.stage_post, project=project, version=version),
+        defaults={"package": project, "version": version},
+    )
+    block.append(form_html)
+
+    return await template.blank("Record Staging Distribution", content=block.collect())
```

Comment:
We use sentence case for titles, [as mentioned in our code conventions](https://release-test.apache.org/docs/code-conventions).